test/test_document.md

```diff
@@ -0,0 +1,125 @@
+# Test data
+## Set of most important tests
+This cases were chosen as minimal set for iteratively check, that program works correctly.
+### First test, most simple. 
+Checks that program can work with one object and such polygon, where its clearly seen, can we plase, or not. 
+![test 0](/test/img/0.jpg "test 0")
+Answer: true
+
+![test 1](/test/img/1.jpg "test 1")
+Answer: false
+
+
+### Precision test
+There are two similar polygons(triangles) and their size is is close to the corner case of bounding triangle of object.
+![test 2](/test/img/2.jpg "test 2")
+Answer: true
+
+![test 3](/test/img/3.jpg "test 3")
+Answer: false
+
+
+### Two convex objects
+Checks that program can work with two convex object.
+For first "two convex" test i choose polygon, for which there is only one correct placement.
+![test 4](/test/img/4.jpg "test 4")
+Answer: true
+
+For second "two convex" test i choose polygon, in which we can place both of objects one at time. And by area this polygon is larger than in first "two convex" test, but both of objects at the same time can't be placed here.
+![test 5](/test/img/5.jpg "test 5")
+Answer: false
+
+
+### Convex and non-convex objects
+Convex and non-convex objects can be plased only with usage of recess of non convex.
+![test 6](/test/img/6.jpg "test 6")
+Answer: true
+
+
+### Handdraw
+Checks that polygon drowen by hand can be scanned and handled correctly.
+![test 7](/test/img/7.jpg "test 7")
+Answer: true
+
+
+### Two non-convex objects
+Two non-convex objects can be plased only with usage of recesses both of non convex.
+![test 8](/test/img/8.jpg "test 8")
+Answer: true
+
+Two non-convex objects can't be plased, but both of them can be placed one at time.
+![test 9](/test/img/9.jpg "test 9")
+Answer: false
+
+
+### Stress test
+Stress test with a lot of objects(all) and wery carefully placement. For true answer i attach picture-proof, because its hard to believe that all objeckts really can be placed on A4.
+![test 10](/test/img/10.jpg "test 10")
+![test 10 proof](/item_set/a.jpg "test 10 proof")
+Answer: true
+
+![test 11](/test/img/11.jpg "test 11")
+Answer: false
+
+
+### Corner cases
```

Comment:
All test cases seem to be representative. I think it will also be nice to see how the program can deal with low resolution or blurred pictures,